tests/test_commands.py

```diff
@@ -0,0 +1,72 @@
+from io import StringIO
+from unittest.mock import patch
+
+from django.core.management import CommandError, call_command
+from django.test import TestCase
+
+import pytest
+
+from digid_eherkenning.models import DigidConfiguration, EherkenningConfiguration
+
+from .conftest import DIGID_TEST_METADATA_FILE_SLO_POST, EHERKENNING_TEST_METADATA_FILE
+
+
+@pytest.mark.usefixtures("digid_config_defaults", "temp_private_root")
+class UpdateStoredMetadataTests(TestCase):
+    @patch(
+        "onelogin.saml2.idp_metadata_parser.OneLogin_Saml2_IdPMetadataParser.get_metadata"
+    )
+    def test_command_triggers_xml_fetching_when_digid(self, get_matadata):
+        config = DigidConfiguration.get_solo()
+
+        with DIGID_TEST_METADATA_FILE_SLO_POST.open("rb") as metadata_file:
+            metadata_content = metadata_file.read().decode("utf-8")
+            get_matadata.return_value = metadata_content
+            config.metadata_file_source = (
+                "https://was-preprod1.digid.nl/saml/idp/metadata"
+            )
+            config.save()
+
+            call_command("update_stored_metadata", "digid")
```

Comment:
how are you sure that the `get_metadata` was called because of the management command and not because of the `save()` method call? :-)

Reply:
Yes, I missed that, thanks.